src/components/Header/Logo.jsx

```diff
@@ -0,0 +1,150 @@
+import React from "react";
+const Logo = () => {
+  return (
+    <div className="nav-logo nav-section-items">
+      {/* <Link to="/" className="link-no-style" > */}
+      {/* <a className="link-no-style" href="/"> */}
```

Comment:
suggestion: remove commented code

Reply:
This will be fixed in next PR. This file would be removed.